service/github/diff.go

```diff
@@ -0,0 +1,79 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+
+	"github.com/google/go-github/v60/github"
+	"github.com/reviewdog/reviewdog"
+)
+
+var _ reviewdog.DiffService = (*PullRequestDiffService)(nil)
+
+type PullRequestDiffService struct {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported type PullRequestDiffService should have comment or be unexported
<!-- __reviewdog__:ChA3OGQxMmVjMjQyNGY2Y2Y5Eg9nb2xpbnQtY2lyY2xlY2k= -->